docs/pages/management/export-audit-events/datadog.mdx

```diff
@@ -0,0 +1,417 @@
+command.
+
+<Tabs>
+<TabItem label="Self-Hosted" scope={["oss","enterprise"]}>
+```code
+$ tctl auth sign --user=teleport-event-handler --out=identity
+```
+
+This command creates one PEM-encoded file, `identity`. The identity file
+includes both TLS and SSH credentials. The Event Handler plugin uses the SSH
+credentials to connect to the Proxy Service, which establishes a reverse tunnel
+connection to the Auth Service. The plugin uses this reverse tunnel, along with
+your TLS credentials, to connect to the Auth Service's gRPC endpoint.
+
+</TabItem>
+<TabItem label="Teleport Cloud" scope={["cloud"]}>
+```code
+$ tctl auth sign --user=teleport-event-handler --out=identity
+```
+
+This command creates one PEM-encoded file, `identity`. The identity file
+includes both TLS and SSH credentials. The Event Handler plugin uses the SSH
+credentials to connect to the Proxy Service, which establishes a reverse tunnel
+connection to the Auth Service. The plugin uses this reverse tunnel, along with
+your TLS credentials, to connect to the Auth Service's gRPC endpoint.
+
+</TabItem>
+<TabItem label="Helm Chart" scope={["cloud"]}>
+
+If you are planning to use the Helm Chart, you'll need to generate the keys
+with the `file` format, then create a secret in Kubernetes.
+
+Create the identity using the following command:
+
+```code
+$ tctl auth sign --format=file --user=teleport-event-handler --out=identity
+```
+
+Then create the Kubernetes secret:
+
+```code
+$ kubectl create secret generic teleport-event-handler-identity --from-file=auth_id=identity
+```
+
+These commands should result in a PEM-encoded file, `identity`, and a secret
+in Kubernetes with the name `teleport-event-handler-identity`.
+
+</TabItem>
+</Tabs>
+
+## Step 5/6. Install Fluentd output plugin for Datadog
+
+In order for Fluentd to communicate with Datadog, it requires the [Fluentd output
+plugin for Datadog](https://github.com/DataDog/fluent-plugin-datadog). Install
+the plugin on your Fluentd host using either `gem` or the `td-agent`, if installed:
+
+```code
+# Using Gem
+$ gem install fluent-plugin-datadog
+
+# Using td-agent
+$ /usr/sbin/td-agent-gem install fluent-plugin-datadog
+```
+
+<Admonition type="tip" title="Testing Locally?">
+
+If you're running Fluentd in a local docker container for testing, you can adjust
+the entrypoint to an interactive shell as the root user, so you can install the
+plugin before starting Fluentd:
+
+```code
+$ docker run -u $(id -u root):$(id -g root) -p 8888:8888 -v $(pwd):/keys -v \
+$(pwd)/fluent.conf:/fluentd/etc/fluent.conf --entrypoint=/bin/sh -it --tty  fluent/fluentd:edge
+# From the container shell:
+$ gem install fluent-plugin-datadog
+$ fluentd -c /fluentd/etc/fluent.conf
+```
+
+</Admonition>
+
+### Configure Fluentd for Datadog
+
+From the Datadog web UI, generate an API key for Fluentd. From **Organization Settings** -> Access -> API Keys, click on **+ New Key**:
```

Comment:
I didn't want to add a link since it will be different depending on your colo. I navigated to it from the user menu:

![image](https://user-images.githubusercontent.com/2349184/221257575-9d052504-6aa3-48c6-88c0-b3735d00a978.png)

But is "User Menu" the right generic term for that?